compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/SyntheticAccessorLowering.kt

```diff
@@ -394,13 +423,20 @@ private class SyntheticAccessorLowering(val context: JvmBackendContext) : IrElem
             return false
 
         val contextDeclarationContainer = allScopes.lastOrNull { it.irElement is IrDeclarationContainer }?.irElement
+//        val contextDeclarationContainer = allScopes.lastOrNull { it.irElement is IrDeclarationContainer }?.irElement.let {if (it is IrClass && it.origin == JvmLoweredDeclarationOrigin.DEFAULT_IMPLS) it.parent else it} as IrElement
```

Comment:
@ting-yuan Obsolete?

Reply:
@max-kammerer Oops. Thanks for catching this.